app/views/mobile-client-builds-row.html

```diff
@@ -41,6 +41,9 @@
               kind="{{row.apiObject.kind}}"
               stay-on-current-page="true"></delete-link>
           </li>
+          <li ng-if="row.buildConfigsVersion | canI : 'update'">
+            <a ng-href="{{row.apiObject | editYamlURL}}" role="button">Edit YAML</a>
```

Comment:
There's an `edit` action which brings you to a form which edits the pipeline. Is it possible to bring that in here as well? 

Can we move this above the delete action just to be consistent with the action list in the pipelines itself. 
![image](https://user-images.githubusercontent.com/9078522/41099804-858581c2-6a57-11e8-9fc9-0e53379b0c34.png)

Reply:
Ill leave this out for now as we may need to implement our own edit page